src/libraries/System.Text.Json/src/ILLink/ILLink.Suppressions.LibraryBuild.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<linker>
+  <assembly fullname="System.Text.Json">
+    <attribute fullname="System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute">
+      <argument>ILLink</argument>
+      <argument>IL2026</argument>
+      <property name="Scope">member</property>
+      <property name="Target">M:System.Text.Json.Nodes.JsonNode.System#Dynamic#IDynamicMetaObjectProvider#GetMetaObject(System.Linq.Expressions.Expression)</property>
+      <property name="Justification">System.Text.Json's integration with dynamic is not trim compatible. However, there isn't a direct API developers call. Instead they use the 'dynamic' keyword, which gets compiled into calls to Microsoft.CSharp. Microsoft.CSharp looks for IDynamicMetaObjectProvider implementations. Leaving this warning in the product so developers get a warning stating that using dynamic JsonValue code may be broken in trimmed apps.</property>
```

Comment:
Does usage of `dynamic` itself causes a warning (without STJ)?

Reply:
Yes. We marked basically all APIs in `Microsoft.CSharp` as `RequiresUnreferencedCode`.

https://github.com/dotnet/runtime/blob/ffb095ae87f3b4e88b03b21d0d945ed969054258/src/libraries/Microsoft.CSharp/ref/Microsoft.CSharp.cs#L10-L34